src/main/webapp/resources/html/download.html

```diff
@@ -16,8 +16,8 @@ <h3>Via FTP</h3>
                 <tr>
                     <td style="text-align: center; border: 1px #cdd4d2 solid;"><a href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/experiments/atlas-latest-data.tar.gz"><pre>atlas-latest-data.tar.gz</pre></a></td>
                     <td style="text-align: center; border: 1px #cdd4d2 solid;">gzipped tar archive of all Expression Atlas analysis results</td>
-                    <td style="text-align: center; border: 1px #cdd4d2 solid;">2019-03-01</td>
-                    <td style="text-align: center; border: 1px #cdd4d2 solid;">86 GB</td>
+                    <td style="text-align: center; border: 1px #cdd4d2 solid;">2019-06-06</td>
```

Comment:
This file is not in use anymore so we could delete it.

Reply:
We should!